tasks/gen/resource/browser.cr

```diff
@@ -103,6 +103,7 @@ class Gen::Resource::Browser < LuckyTask::Task
   private def display_success_messages
     success_message(resource_name, "./src/models/#{underscored_resource}.cr")
     success_message("Save" + resource_name, "./src/operations/save_#{underscored_resource}.cr")
+    success_message("Delete" + resource_name, "./src/operations/delete_#{underscored_resource}.cr")
```

Comment:
Now that this file is generated, we just need to fix your previous PR 😄 

https://github.com/luckyframework/lucky/blob/master/tasks/gen/templates/resource/actions/%7B%7Bfolder_name%7D%7D/delete.cr.ecr#L4

This file should change from `<%= resource %>::DeleteOperation` to `Delete<%= resource %>`

Reply:
Aaaah now I see it 😄 thanks!